src/Sentry.Bindings.Cocoa/ApiDefinitions.cs

```diff
@@ -416,9 +416,9 @@ interface SentryEnvelopeItemHeader : SentrySerializable
     [Export ("initWithType:length:filenname:contentType:")]
     NativeHandle Constructor (string type, nuint length, string filename, string contentType);
 
-    // -(instancetype _Nonnull)initWithType:(NSString * _Nonnull)type length:(NSUInteger)length contentType:(NSString * _Nonnull)contentType itemCount:(NSNumber * _Nonnull)itemCount;
+    // -(instancetype _Nonnull)initWithType:(NSString * _Nonnull)type length:(NSUInteger)length contentType:(NSString * _Nullable)contentType itemCount:(NSNumber * _Nonnull)itemCount;
     [Export ("initWithType:length:contentType:itemCount:")]
-    NativeHandle Constructor (string type, nuint length, string contentType, NSNumber itemCount);
+    NativeHandle Constructor (string type, nuint length, [NullAllowed] string contentType, NSNumber itemCount);
```

Comment:
Why has this changed?

Reply:
I suppose we are missing dirty-check for `ApiDefinitions.cs`. It was changed to nullable in 8.56.1:
- #6186